examples/soap/excluir_fornecedor.py

```diff
@@ -0,0 +1,41 @@
+from fortes_webservice_wrapper.wrapper.fortes_ws import FortesWrapper
+
+fornecedor = {
+    "Codigo": "",
+    "Nome": "Ana do Imobanco",
+    "CNPJCPF": "31510515020",
+    "CEP": "",
+    "Logradouro": "",
+    "Bairro": "",
+    "Numero": "",
+    "Complemento": "",
+    "UF": "",
+    "Cidade": "",
+    "DDD": "",
+    "Fone": "",
+    "Fax": "",
+    "Email": "",
+    "Contato": "",
+    "Site": "",
+    "RG": "",
+    "IE": "",
+    "Grupo": "",
+    "Receita": "",
+    "IM": "",
+    "Referencia": "",
+    "FinalidadeTED": "",
+    "FinalidadeDOC": "",
+    "Celular": "",
+    "Leiaute": "",
+    "Banco": "",
+    "BancoNumero": "",
+    "Agencia": "",
+    "CC": "",
+    "DVAg": "",
+    "DVCC": "",
+    "ServicoPublico": "",
+    "ICMS": "",
+    "IPI": "",
+}
+
+r = FortesWrapper().excluir_fornecedor_com_json(fornecedor)
```

Comment:
Talvez mudar o nome para ficar no padrão dos outros?

```suggestion
r = FortesWrapper().excluir_fornecedor(fornecedor)
```